lib/index.d.ts

```diff
@@ -69,9 +69,9 @@ declare namespace Celebrate {
     /**
      * The standard error used by Celebrate
      */
-    declare class CelebrateError {
-        constructor(error: ValidationError, segment: Segments, opts?: { celebrated: boolean }) {}
+    class CelebrateError {
```

Comment:
So `CelebrateError` is actually a factory function, so it's technically not a class, just a function that you call which *creates* a `CelebrateError`. That was something I discovered myself locally. If you could make that change too, that'd be awesome.

Reply:
It was not intentional that you don't have access. The checkbox to allow changes from maintainers is ticked. What else do I have to do to allow access to the branch for you?

Reply:
This wouldn't work because the name of the Function and the type `function CelebrateError(error: ValidationError, segment: Segments, opts?: { celebrated: boolean }) : CelebrateError`would refer to the same thing, not one to the function and one to the type or do you mean something like this `function CelebrateError(error: ValidationError, segment: Segments, opts?: { celebrated: boolean }) : Error`